py/legacypipe/queue-calibs.py

```diff
@@ -427,6 +427,9 @@ def main():
     elif opt.region == 'dr8-test-deep2-egs':
         rlo, rhi = 213, 216.5
         dlo, dhi = 52, 54
+    elif opt.region == 'dr8-overlap':
+        rlo, rhi = 132, 140.5,
```

Comment:
Since you have this branch handy, @geordie666, can you call this region "dr8-test-overlap"? After this change please feel free to self-merge, thanks!